samples/subsys/usb/mass/prj_nrf52840_pca10056.conf

```diff
@@ -3,10 +3,24 @@ CONFIG_GPIO=y
 CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
 CONFIG_USB_DEVICE_PRODUCT="Zephyr MSC sample"
+CONFIG_USB_MASS_STORAGE=y
 CONFIG_LOG=y
+
 CONFIG_USB_DRIVER_LOG_LEVEL_ERR=y
-CONFIG_USB_MASS_STORAGE=y
 CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
 CONFIG_USB_MASS_STORAGE_LOG_LEVEL_ERR=y
+CONFIG_DISK_LOG_LEVEL_ERR=y
+
 CONFIG_DISK_ACCESS_RAM=y
-CONFIG_MASS_STORAGE_DISK_NAME="RAM"
+
+CONFIG_MASS_STORAGE_DISK_NAME="NAND"
+CONFIG_DISK_ACCESS_FLASH=y
+CONFIG_DISK_FLASH_DEV_NAME="NRF_FLASH_DRV_NAME"
+CONFIG_DISK_FLASH_START=0xf8000
+CONFIG_DISK_FLASH_MAX_RW_SIZE=4
```

Comment:
CONFIG_DISK_FLASH_MAX_RW_SIZE can be safely increased. We have no limit for this value - It is possible to wite/read whole flash at once. Try use one of 256/1014/4096

Reply:
Changed to 4096.